Source/BlueprintSearchVSExtension/Source/Commands/CommandHandlers/ExecuteSearchHandler.cs

```diff
@@ -6,24 +6,32 @@
 using Newtonsoft.Json;
 using System.Collections.Generic;
 using System.IO;
-using System.Windows;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace BlueprintSearch.Commands.CommandHandlers
 {
 	public class ExecuteSearchHandler
 	{
+		public List<BlueprintJsonObject> Results = new List<BlueprintJsonObject>();
+
+		public List<string> ErrorMessages = new List<string>();
+
 		private const string CommandLineArguments = "-NoShaderCompile -SILENT -run=FiB";
 
-		public List<BlueprintJsonObject> MakeSearch(string InSearchValue)
+		public async Task MakeSearchAsync(string InSearchValue, CancellationToken cancellationToken)
```

Comment:
Well, the coding standards are a bit all over the place in this project I can see. But can this cancellationToken have a capital C please so at least this file is consistent.